resources/js/main.js

```diff
@@ -3,9 +3,8 @@ import Vue from 'vue'
 import App from './App.vue'
 import router from './router'
 import store from './store/index.js'
-import './registerServiceWorker'
-import './../sass/app.scss'
 import VueProgress from 'vue-progress-path'
+import './../sass/app.scss'
```

Comment:
Great! While you're at it, can you please also remove this import statement!? We don't need it because we are referencing the compiled app.css file in welcome.blade.php